.github/workflows/self-scheduled-caller.yml

```diff
@@ -0,0 +1,65 @@
+name: Self-hosted runner (scheduled)
+
+
+on:
+  repository_dispatch:
+  schedule:
+    - cron: "17 2 * * *"
+  push:
+    branches:
+      - run_scheduled_ci*
+
+jobs:
+  model-ci:
+    name: Model CI
+    uses: ./.github/workflows/self-scheduled.yml
+    with:
+      job: run_tests_gpu
+      # See the comment for `ENV_NAME_FOR_CI_SLACK_REPORT_CHANNEL_ID` in `.github/workflows/slack-report.yml`.
+      env_name_for_slack_report_channel: CI_SLACK_CHANNEL_DUMMY_TESTS
```

Comment:
- the input `job` is used to control which job defined in `self-scheduled.yml` is to be run
  - (and used in  `notification_service.py` to control some reporting logic)
- the input `env_name_for_slack_report_channel` is used to control which channel to report for a specific job